### PR TITLE
fetch asset data on details page and update container state

### LINKED ISSUE #671: have asset details page fetch and update container's state
potentially move revoke/delete actions into details page, so user would have to hit them before doing any actions

### PR DESCRIPTION
**Pull request by laholstege**
closes #671

### CODE REVIEW

File: Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx [hunk 1]
```diff
@@ -70,13 +70,9 @@ export default class EquipmentContainer extends React.Component<IProps, IState>
...
         if (!!this.props.person) {
-            equipmentFetchUrl = `/api/${this.context.team.slug}/equipment/listassigned?personid=${
-                this.props.person.id
-            }`;
+            equipmentFetchUrl = `/api/${this.context.team.slug}/equipment/listassigned?personid=${this.props.person.id}`;
         } else if (!!this.props.space) {
-            equipmentFetchUrl = `/api/${
-                this.context.team.slug
-            }/equipment/getEquipmentInSpace?spaceId=${this.props.space.id}`;
+            equipmentFetchUrl = `/api/${this.context.team.slug}/equipment/getEquipmentInSpace?spaceId=${this.props.space.id}`;
```
Comment (srkirkland) [inline Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx]: your formatter cannot make up its mind

Comment (laholstege) [reply Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx]: it just formats when i save 🤷‍♀

File: Keas.Mvc/Controllers/Api/KeySerialsController.cs [hunk 2]
```diff
@@ -138,18 +152,18 @@ public async Task<IActionResult> Create([FromBody] CreateKeySerialViewModel mode
...
-                Number   = model.Number.Trim(),
-                Notes    = model.Notes, 
-                Status   = model.Status ?? "Active",                
+                KeyId = key.Id,
+                Key = key,
+                TeamId = key.TeamId,
+                Name = model.Number.Trim(),
+                Number = model.Number.Trim(),
+                Notes = model.Notes,
+                Status = model.Status ?? "Active",
```
Comment (srkirkland) [inline Keas.Mvc/Controllers/Api/KeySerialsController.cs]: haha John is barely gone and then you do this to him!

Comment (jSylvestre) [reply Keas.Mvc/Controllers/Api/KeySerialsController.cs]: Status should not be null (I fixed that)

File: Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx [hunk 3]
```diff
@@ -60,7 +60,7 @@ export default class EquipmentContainer extends React.Component<IProps, IState>
             equipmentProtectionLevels: ["P1", "P2", "P3", "P4"],
             equipmentProtectionFilters: [],
             equipmentAvailabilityLevels: ["A1", "A2", "A3", "A4"],
-            equipmentAvailabilityFilters: [],
+            equipmentAvailabilityFilters: []
```
Comment (jSylvestre) [inline Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx]: When you remove this, it looks like you change it when you add another line

**Pull request merged**